### DISCUSSION TITLE
English (UK): Settings > "Customize" or "Customise" tab

### DISCUSSION THREAD

**Discussion by chip33** [Hofknecht/SystemTrayMenu#402] [Q&A] [answered]
Hi,
this is not that big of an issue so I am mentioning it in discussions rather than making a feature request but I noticed the Customise tab in Settings uses the English (US) spelling Customize while my system is set to English (UK).
Just thought I would mention it in case it is easy to fix.
Thanks :)

**Comment by Hofknecht** [comment]
@chip33 
Thanks for the feedback!

we use this tool here AutoResxTranslator
https://github.com/Hofknecht/SystemTrayMenu/issues/388

here we get automatic translations from google.
on google translate I can't find UK English on the first try.
Of course, it is theoretically possible to do this.
We make a note and look at the topic again.

**Comment by Hofknecht** [comment, marked as answer]
Currently with AutoResxTranslator i can not automatically translate UK English
Also with google i am not able to to translate into UK English
If you want to you can copy lang.resx, translate it into UK English, we can add it then.
We would then only have to maintain it manually, i.e. you would have to check from time to time whether the English translations that we add fit

**Comment by chip33** [comment]
@Hofknecht

Hi, I just saw this reply.

Only "Customize" and several uses of "Color" needed changing for en-GB.

I created a new Feature Request  [Add English GB language support #414](https://github.com/Hofknecht/SystemTrayMenu/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature%5D) and posted the lang.en-GB.resx file there.

A few things could be changed for both English International and GB but I did not correct them myself, as they are just suggestions.

If you agree with any of these I can alter lang.resx (and lang.en-GB.resx) and post it here.

"Mouse location (Above Taskbar icon)" should be "Mouse location (above Taskbar icon)"

"Read the FAQ and then choose a root directory for the SystemTrayMenu." is more correct as
"Read the FAQ and then choose a root directory for SystemTrayMenu."

Settings -> Expert -> "Stays open" may be clearer as "Menu stays open" ?

Settings -> Customize -> "Show number of items below" may be clearer as "Show number of items" ?

"If the focus is lost and if the mouse is still on the menu" could be simplified to 
"If the focus is lost and the mouse is still on the menu"

I believe it would still be clear if the following 2 statements were also simplified, as the checkboxes above them visually indicate when they are ticked:

"Milliseconds until the menu closes if in this case the mouse then leaves the menu" could be simplified to "Milliseconds until the menu closes if the mouse then leaves the menu"

"Milliseconds until the menu closes if in this case the menu is not reactivated" could be simplified to "Milliseconds until the menu closes if the menu is not reactivated"

`"Configuration file and log file"  is better as "Configuration and log files" then,
"Save configuration in application directory" could be "Save configuration file in application directory" to match "Save log file in application directory" 


 "Copy row item via drag drop"  should probably be "Copy row item via drag and drop"

  "Custom (drag it to the appropriate place)" could be "Custom (drag it to the appropriate position)"

**Comment by Hofknecht** [comment]
@chip33 
We accept all suggestions, thank you for all your work. I commit it with #414

**Reply by chip33** [reply]
Thank you , works very well

**Reply by chip33** [reply]
@Hofknecht 

Apologies for not including (or noticing) these before but if you agree, in Settings -> Size and location:

"% Size" could be "Application size in percent"
"Size of the icons in percent" could be "Icon size in percent"
"Pixels maximum menu width" could be "Maximum menu width in pixels"
"Pixels maximum menu height" could be "Maximum menu height in pixels"
"Offset by pixels" could be "Offset in pixels"

**Comment by Hofknecht** [comment]
@chip33 
Thanks for the further improvements!
committed with version 1.3.0.5
If you have further improvements, we are happy to take them!
We like the improved texts!

**Comment by chip33** [comment]
@Hofknecht 
I am not sure how well it would work with automatic translations but "Show number of items" looks good as "Show item count"

**Comment by Hofknecht** [comment]
@chip33 
you probably mean this node in resx file
```
<data name="Show count of elements" xml:space="preserve">
    <value>Show number of items</value>
  </data>
```
"Show count of elements" is the key.
"Show number of items" is the value for english and for e-GB.

we translate from key into other languages. 
We did this for the reason that e.g. in German the translation of "Show number of items" is "Anzahl der Artikel anzeigen", which is wrong meaning kind of "Show number of articles". The translation of "Show count of elements" is "Anzahl der Elemente anzeigen" which is "Show number of items" in english. 

We think that should fit, if you have further hints here, you're welcome!

**Reply by chip33** [reply]
Smiles, thank you for the great explanation. :)
